python-microservices/pizza-service/config.properties

```diff
@@ -0,0 +1,13 @@
+[kafka_client]
+bootstrap.servers=<Kafka server or Confluent Cloud endpoint>
+security.protocol=SASL_SSL
+sasl.mechanisms=PLAIN
+sasl.username=<Confluent Cloud API Key>
+sasl.password=<Confluent Cloud API Secret>
+
+[consumer]
+auto.offset.reset=earliest
+group.id=pizza_shop
+enable.auto.commit=true
```

Comment:
When the service commits and then restarts, do you lose all the orders in the warmer?